src/Chapter1.hs

```diff
@@ -541,8 +543,12 @@ value after "=" where the condition is true.
 
 Casual reminder about adding top-level type signatures for all functions :)
 -}
+mid :: Int -> Int -> Int -> Int
+mid x y z
+  | min x y == min y z = min x z
+  | otherwise = max (min x y) (min y z)
```

Comment:
Creative solution! 🧑‍🎨 
As you reuse some of the `max` and `min` values, it makes sense to move them into separate variables in the `where` clause, e.g. `minXy = min x y` , etc. 🙂